ios/INVRegionMonitor.m

```diff
@@ -138,13 +138,19 @@ - (BOOL)_isRegionMonitoringPossible:(CLAuthorizationStatus)status {
     );
 }
 
-- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit {
+- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit locationManager:(CLLocationManager *)locationManager {
```

Comment:
`locationManager` is available as local property. Why do you want to pass it along?